torch/_export/__init__.py

```diff
@@ -262,6 +269,8 @@ def _export(
     constraints: Optional[List[Constraint]] = None,
     *,
     strict: bool = True,
+    decomp_table: Optional[Dict[str, Callable]] = None,
+    pre_dispatch: bool = False,
```

Comment:
Seems like these two arg aren't being used?

Reply:
it is actually used here: https://github.com/pytorch/pytorch/pull/115210/files#diff-7c238fb036dbd2844e5ecdcb28e35f5cc3eb3a664f212f2d9a066e9c6ede7019R182

Reply:
nvm haha. I thought this was `torch/export/__init__.py`

Reply:
wait sorry, I'm still confused by these two arguments. They're not being used in _export though?